licence.json

```diff
@@ -0,0 +1,79 @@
+{
+  "id": 99999,
+  "type": "STD | HDC",
+  "subType": "AP | PSS | AP+PSS",
+  "policyVersion": "2.1",
+  "version": "1.4",
+  "statusCode": "ACTIVE",
+  "nomsId": "A9999AA",
+  "bookingId": 989898,
+  "crn": "X12444",
+  "approvedByUsername": "X33221",
+  "approvedDate": "2023-09-27T16:59:04.886Z",
```

Comment:
Could we not just provide the date (or is the timestamp necessary)

Reply:
Think it makes sense to have the timestamp as well - as these refer to events that happen at a specific time. 

But would could maybe call them `*DateTime`, e.g: `ApprovedDateTime`?

Reply:
We'll need to add something to the docs to explain that licences approved before X will not have an approved time.